nomos-da/kzgrs/src/kzg.rs

```diff
@@ -132,4 +136,47 @@ mod test {
             }
         }
     }
+
+    #[test]
+    #[cfg(feature = "parallel")]
+    fn parallelized_generate_proof_and_validate() {
+        let mut bytes: [u8; 310] = [0; 310];
+        let mut rng = thread_rng();
+        bytes.try_fill(&mut rng).unwrap();
+        let (eval, poly) = bytes_to_polynomial::<31>(&bytes, *DOMAIN).unwrap();
```

Comment:
I think we have a constant for those `31` it would be better to use it actually.